aea/protocols/base.py

```diff
@@ -63,8 +63,8 @@ def __init__(self, body: Optional[Dict] = None, **kwargs):
         """
         self._to = None  # type: Optional[Address]
         self._sender = None  # type: Optional[Address]
-        self._body = copy(body) if body else {}  # type: Dict[str, Any]
-        self._body.update(kwargs)
+        self.__body = copy(_body) if _body else {}  # type: Dict[str, Any]
+        self.__body.update(kwargs)
```

Comment:
the leading double underscore has a special meaning in Python. I think it is OK to store `_body`

Reply:
https://dbader.org/blog/meaning-of-underscores-in-python

Reply:
That's what I'm after (or at least don't mind the properties it provides)